documentation/adr/0008-pace-syntax-guiding-principles.md

```diff
@@ -0,0 +1,44 @@
+[<-previous](0007-developer-scripts-storage-location.md) | next->
+
+# 8. PACE Syntax Guiding Principles
+
+Date: 2020-Oct-19
+
+## Status
+
+Proposed
+
+
+## Context
+
+Whilst the core framework of PACE is the Horace (and Herbert) projects,
+PACE involves many other codes, including [Euphonic](https://github.com/pace-neutrons/Euphonic),
+[Brille](https://github.com/brille/brille) and [SpinW](https://spinw.org).
+The way in which these programs interact with each other is presently unclear.
+Furthermore the programs are written in several different languages.
+In this ADR we propose some guiding principles in the design of the interfaces
+between the programs or components which make up PACE.
```

Comment:
I've now caught up with the location of the minutes of the meeting (opening comment, above) and it would be good just to have that location recorded in the ADR.

Additional thought: is github the best place to put these documents? We have the sharepoint, which may be more appropriate.